lonestar/mining/include/lonestarmine.h

```diff
@@ -21,7 +21,7 @@ static cll::opt<unsigned>
             cll::init(1));
 static cll::opt<std::string>
     pattern_filename("p",
-                     cll::desc("<pattern graph filename: unsymmetrized graph>"),
+                     cll::desc("<pattern graph filename: symmetrized graph>"),
```

Comment:
Are you sure the pattern file name is supposed to be symmetrized? I would make sure with someone familiar with the mining code.

Reply:
Yes. Xuhao confirmed this.

Reply:
Cool, thanks.